Bazel build failure

The commit 92ca5e8 moves `llvm/tools/llvm-ir2vec/llvm-ir2vec.h` to `llvm/tools/llvm-ir2vec/utils/utils.h`.
However, the `utils/bazel/llvm-project-overlay/llvm/BUILD.bazel:6630` relies on `tools/llvm-ir2vec/*.h`, which will cause an error when allow_empty is set to False, as shown below：

```
Run bazelisk test --config=ci --sandbox_base="" \
  bazelisk test --config=ci --sandbox_base="" \
    --remote_cache=https://storage.googleapis.com/$CACHE_GCS_BUCKET-bazel \
    --google_default_credentials \
    @llvm-project//... //...
  shell: /usr/bin/bash -e {0}
2026/01/17 14:14:06 Downloading https://releases.bazel.build/8.4.2/release/bazel-8.4.2-linux-x86_64...
Extracting Bazel installation...
Starting local Bazel server (8.4.2) and connecting to it...
INFO: Invocation ID: 3e97e4e7-75ec-4120-9200-ad8ad65ca4ff
Computing main repo mapping: 
Computing main repo mapping: 
Computing main repo mapping: 
Computing main repo mapping: 
Computing main repo mapping: 
ERROR: Traceback (most recent call last):
	File "/home/gha/.cache/bazel/_bazel_gha/7e8ec8c8c4975cf00cc8d263f2c10e24/external/+_repo_rules+llvm-project/llvm/BUILD.bazel", line 6628, column 16, in <toplevel>
		srcs = glob([
Error in glob: glob pattern 'tools/llvm-ir2vec/*.h' didn't match anything, but allow_empty is set to False (the default value of allow_empty can be set with --incompatible_disallow_empty_glob).
WARNING: Target pattern parsing failed.
ERROR: Skipping '@llvm-project//llvm:pfm': Error evaluating '@llvm-project//llvm:pfm': error loading package '@@+_repo_rules+llvm-project//llvm': Package 'llvm' contains errors
ERROR: @llvm-project//llvm:pfm :: Error loading option @llvm-project//llvm:pfm: Error evaluating '@llvm-project//llvm:pfm': error loading package '@@+_repo_rules+llvm-project//llvm': Package 'llvm' contains errors
Error: Process completed with exit code 2.
```

cc @nishant-sachdeva

--- a/utils/bazel/llvm-project-overlay/llvm/BUILD.bazel
+++ b/utils/bazel/llvm-project-overlay/llvm/BUILD.bazel
@@ -6627,7 +6627,7 @@ cc_binary(
     testonly = True,
     srcs = glob([
         "tools/llvm-ir2vec/*.cpp",
-        "tools/llvm-ir2vec/*.h",
+        "tools/llvm-ir2vec/utils/*.h",
     ]),
     copts = llvm_copts,
     stamp = 0,

